src/role-access/containers/exclusion/describe-exclusion/describe-exclusion.component.html

```diff
@@ -1,3 +1,24 @@
-<p>
-  describe-exclusion works!
-</p>
+<exui-error-message *ngIf="error" [error]="error"></exui-error-message>
+<div [formGroup]="formGroup"
+     class="govuk-form-group"
+     [ngClass]="{'form-group-error': !formGroup.get(controlName).valid && formGroup.get(controlName).touched && submitted}">
+  <h1 class="govuk-heading-l">
+    <span class="govuk-caption-l">{{title}}</span>
+    {{description}}
+  </h1>
+  <span id="more-detail-error" class="govuk-error-message"
+        *ngIf="!formGroup.get(controlName).valid && formGroup.get(controlName).touched && submitted">
+    <span class="govuk-visually-hidden">Error:</span> Enter exclusion
+  </span>
+  <fieldset class="govuk-fieldset">
+    <textarea
+      [ngClass]="{'govuk-textarea--error': !formGroup.get(controlName).valid && formGroup.get(controlName).touched && submitted}"
+      class="govuk-textarea govuk-!-width-one-half govuk-!-margin-bottom-0"
+      id="exclusion-description"
+      formControlName="text"
+      name="exclusion-description"
+      rows="5"
+      aria-describedby="exclusion-description"
```

Comment:
it seems this aria-describedby not set correclty, it's binding itself but not other element.
see usage on:
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-describedby_attribute, it seems it's more proper to use aria-label="enter exclusion description" here.

Reply:
@rayliangatos  fixed it now